static/src/stylesheets/module/external/_open-module.scss

```diff
@@ -1,19 +1,19 @@
 .open {
     margin-bottom: $gs-baseline;
-    border-top: 2px solid;
+    border-top: 1px solid;
     border-bottom: 1px solid colour(neutral-4) !important;
 }
 
 h2.open__head { // element selector here in order to override .from-content-api style
+    @include fs-headline(3);
+    font-weight: 500;
     color: colour(neutral-2);
-    @include fs-header(2);
     margin-top: 2px;
     margin-bottom: $gs-baseline !important;
-    line-height: 22px;
 
-    @include mq(tablet) {
-        @include fs-header(3);
-        line-height: 24px;
+    @include mq($until: tablet) {
+        @include font-size(18, 22);
+        font-weight: 500;
     }
```

Comment:
logic seems slightly backwards (and `font-weight` is duplicated). This should be a `mq(tablet)`, rather than `until`. Move this `font-size` into the containing selector (with `f-headline`), and have a `fs-headline(3, $size-only: true)` in here